draft-ion.adoc

```diff
@@ -105,16 +120,13 @@ A single Ion instance resource without metadata is simply a standard JSON object
 }
 ----
 
-A single Ion instance resource with metadata is a JSON object that has a direct
-child `meta` name/value pair in addition to its other member name/value pairs:
+An Ion _Resource_ is a JSON object with a Metadata member named `meta` in addition to one or more other members (name/value pairs):
```

Comment:
Can't this be zero or more members. May not be useful or typical, but should be valid. May just be a resource that is a link to some other resource for whatever reason, for example.